Improve documentation of gateway addresses (#5024)
camunda/camunda-docs: versioned_docs/version-8.6/apis-tools/spring-zeebe-sdk/configuration.md

#### REST over gRPC

> If true, the Zeebe Client will use REST instead of gRPC whenever possible to talk to the Zeebe Gateway:

Reviewer: ```suggestion
If true, the Zeebe Client will use REST instead of gRPC whenever possible to communicate with the Zeebe Gateway:
```
Rewording suggestion

      grpc-address: http://localhost:26500
```

> Make sure to add the scheme `http://` to the URL. Otherwise you will get a `java.lang.NullPointerException: target`.

Reviewer 2: @camunda/tech-writers this could be wordsmithed a bit and turned into an admonition.

> Make sure to add the scheme `http://` to the URL. Otherwise you will get a `java.lang.NullPointerException: target`.

Reviewer: ```suggestion
:::note
You must add the `http://` scheme to the URL to avoid a `java.lang.NullPointerException: target` error.
:::
```

#### REST address

> Define address of the [Camunda REST API](../camunda-api-rest/camunda-api-rest-overview.md) exposed by the [Zeebe Gateway](../../self-managed/zeebe-deployment/zeebe-gateway/zeebe-gateway-overview.md):

Reviewer 2: ```suggestion
Define address of the [Camunda 8 REST API](../camunda-api-rest/camunda-api-rest-overview.md) exposed by the [Zeebe Gateway](../../self-managed/zeebe-deployment/zeebe-gateway/zeebe-gateway-overview.md):
```

```yaml
camunda:
  client:
    zeebe:
      rest-address: http://localhost:8080
```

> Make sure to add the scheme `http://` to the URL.

Reviewer: ```suggestion
:::note
You must add the `http://` scheme to the URL.
:::
```
Rewording suggestion - not sure if the error warning is required for this one?
